client-app/src/Components/DonatedItemsList.tsx

```diff
@@ -362,70 +362,63 @@ const DonatedItemsList: React.FC = () => {
             </div>
 
             <table className="item-list">
-                <thead>
-                    <tr>
-                        <th>S.No</th>
-                        <th>Item ID</th>
-                        <th>Item Name</th>
-                        <th>Status</th>
-                        <th>Donation Date</th>
-                        <th>Barcode</th>
-                        {assignProgramClicked && <th>Select</th>}
-                    </tr>
-                </thead>
-                <tbody>
-                    {(filteredItems.length > 0
-                        ? filteredItems
-                        : donatedItems
-                    ).map((item, index) => (
-                        <tr key={item.id}>
-                            <td>{index + 1}</td>
-                            <td>
-                                <Link
-                                    to={`/donations/${item.id}`}
-                                    state={{ itemInfo: item }}
-                                >
-                                    {item.id}
-                                </Link>
-                            </td>
-                            <td>{item.itemType}</td>
-                            <td>{item.currentStatus}</td>
-                            <td>
-                                {new Date(
-                                    item.dateDonated,
-                                ).toLocaleDateString()}
-                            </td>
-                            <td>
-                                <div
-                                    onClick={() => handleBarcodeClick(item.id)}
-                                >
-                                    <div id={`barcode-${item.id}`}>
-                                        <Barcode value={item.id.toString()} />
-                                    </div>
-                                    <button
-                                        onClick={() => downloadBarcode(item.id)}
-                                    >
-                                        Download Barcode
-                                    </button>
-                                </div>
-                            </td>
-                            {assignProgramClicked && (
-                                <td>
-                                    <input
-                                        type="checkbox"
-                                        checked={selectedItems.includes(
-                                            item.id,
-                                        )}
-                                        onChange={() =>
-                                            handleCheckboxChange(item.id)
-                                        }
-                                    />
-                                </td>
-                            )}
-                        </tr>
-                    ))}
-                </tbody>
-            </table>
+    <thead>
+        <tr>
```

Comment:
It seems these are the trivial changes with some indentation removed. Prettier should apply the formatting updates to your code. Please run prettier write on your code